f/ffnvcodec-headers/stone.yaml

```diff
@@ -0,0 +1,21 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name       : ffnvcodec-headers
```

Comment:
I have a slight preference for naming this package `nv-codec-headers`

Reply:
Fixed